shub_image/build.py

```diff
@@ -48,8 +47,7 @@ def build_cmd(target, version):
         raise shub_exceptions.BadParameterException(
             'Dockerfile is not found, please use shub-image init cmd')
     is_built = False
-    for line in client.build(path=project_dir, tag=image_name):
-        data = json.loads(line)
+    for data in client.build(path=project_dir, tag=image_name, decode=True):
```

Comment:
`decode` param was added to `build` command in docker-py 1.3.0, to `push`/`pull` in 1.8.0, but works fine only since 1.10.0 version due to [the bug](https://github.com/docker/docker-py/blob/master/docs/change_log.md#1100); it explains picking docker-py>=1.10.0 version in extras.